_rules/device-motion-disabled-c249d5.md

```diff
@@ -30,19 +30,22 @@ The rule applies to an [HTML document][] with an associated [Window object][] th
 
 ## Expectation 1
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] to prevent the outcome of the event is available in the same [web page][], or in another [web page][] that is [linked][hyperlink] from the [web page][] of the test target.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target there exists a set of one or more [instruments][instrument] to prevent any results of the event within a 10 minute time span of the [event firing][].
 
 **Note:** The same [instrument][] can be used to disable more than one event.
 
-**Note:** Preventing the outcome of the event can be done in multiple ways (e.g. removing the event listener; handling the event in a different manner; ...) but the way in which it is done is not relevant for this rule.
+**Note:** Preventing results of the event can be done in multiple ways (e.g. removing the event listener; handling the event in a different manner; ...) but the way in which it is done is not relevant for this rule.
+
+**Note:** The 10 minute time span is an arbitrary limit which is not included in WCAG. Results that happen after this period will not fail this rule but may nonetheless fail [Success Criterion 2.5.4: Motion Actuation][sc 2.5.4]. The accessibility problem tends to be less severe for longer time periods, and without a time limit, testing this rule consistently would be impractical.
 
 ## Expectation 2
 
-The [instrument][] is [visible][].
+For each [instrument][] in the set of [instruments][instrument] from Expectation 1, one of the following is true:
 
-## Expectation 3
+- the [instrument][] is in the same [web page][] of the test target; or
+- instructions for the user to locate the [instrument][] are in the same [web page][] of the test target.
```

Comment:
I don't think WCAG requires that there are instructions on every page.

Reply:
@WilcoFiers Indeed. I was just waiting for the definition of 'clearly labeled location' from #960 
Updating this expectation to use that definition.